### PR TITLE
www/c-icap: Change 50-c-icap script.

### PR DESCRIPTION
**Pull request by AndyX90**
The icap-service needs to be restarted after the start of clamav otherwise c-icap runs but the clamd module won't register/connect to clamav properly. Also in its current form it throws "Error in start script '50-c-icap'".

```
...
 clamd_connect: Can not connect to clamd server on 127.0.0.1:3310!
 clamd_init: Error while sending command to clamd server
 Registry 'virus_scan::engines' does not exist!
...
```

### CODE REVIEW

File: www/c-icap/src/etc/rc.syshook.d/start/50-c-icap [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-# start again to fix race with clamav (no need to restart)
-/usr/local/etc/rc.d/c-icap start
+# restart service to make sure clamav is started
```
Comment (fichtner) [inline www/c-icap/src/etc/rc.syshook.d/start/50-c-icap:3]: The comment is wrong now

Comment (AndyX90) [reply www/c-icap/src/etc/rc.syshook.d/start/50-c-icap]: @fichtner Sorry, it was too late yesterday😅

File: www/c-icap/src/etc/rc.syshook.d/start/50-c-icap [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
-# start again to fix race with clamav (no need to restart)
-/usr/local/etc/rc.d/c-icap start
+# restart service to make sure c-icap starts after clamav is ready
+/usr/local/etc/rc.d/c-icap restart
```
Comment (fichtner) [inline www/c-icap/src/etc/rc.syshook.d/start/50-c-icap:4]:
```suggestion
/usr/local/etc/rc.d/c-icap restart

```

**Comment by fichtner** [comment]
Merged, thanks!

**Comment by mimugmail** [comment]
Do we need to bump version? Can't see it in current version ...

**Comment by fichtner** [comment]
yep, this is the change for 24.7.12: https://github.com/opnsense/plugins/commit/170a9bee9b

**Pull request merged**